python/lsst/daf/butler/core/uri/__init__.py

```diff
@@ -0,0 +1,5 @@
+# Should only expose ButlerURI
+
+from ._butlerUri import ButlerURI
+__all__ = ("ButlerURI",)
```

Comment:
We don't usually put `__all__` in `__init__.py` since for the way we use `__init__.py` it mostly is for forwarding symbols anyhow. The `__all__` here does not serve any purpose so can be removed.

Reply:
Understood, wasn't sure will fix